src/fsharp/FSharp.Core/FSharp.Core.fsproj

```diff
@@ -1,16 +1,30 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information. -->
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netstandard2.0</TargetFrameworks>
```

Comment:
Do we need a `netstandard2.1` build of FSharp.Core? // cc @KevinRansom

Reply:
Have discussed with Kevin. He had misgivings (due to past hell with this sort of thing on .NET Framework), thought about it a lot, and then we've agreed it's the right thing to do to allow access to ValueTask in FSharp.Core.